.github/workflows/python-ci.yml

```diff
@@ -15,17 +15,30 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # TODO: restore Windows build once we have C++/libtiledbsoma integration supported there
-        os: [ubuntu-22.04, macos-12]
-        # os: [ubuntu-22.04, macos-12, windows-2019]
-        python-version: ['3.7', '3.8', '3.9', '3.10']
-        include:
-          - runs-on: ubuntu-22.04
-            cc: gcc-11
-            cxx: g++-11
-          - runs-on: macos-12
-            cc: gcc-11
-            cxx: g++-11
+        if: github.event_name != "pull_request"
+          # TODO: restore Windows build once we have C++/libtiledbsoma integration supported there
+          os: [ubuntu-22.04, macos-12]
+          # os: [ubuntu-22.04, macos-12, windows-2019]
+          python-version: ['3.7', '3.8', '3.9', '3.10']
+          include:
+            - runs-on: ubuntu-22.04
+              cc: gcc-11
+              cxx: g++-11
+            - runs-on: macos-12
+              cc: gcc-11
+              cxx: g++-11
+        if: github.event_name == "pull_request"
```

Comment:
I don't pretend to fully understand the github workflow eventing semantics, but I would have thought we want to trigger a build per `push` event, rather than `pull_request`, since the latter seems to involve "activity types" like 
- assigned
- unassigned
- labeled
- unlabeled
- opened
- edited
- closed
...

Reply:
Super-good call @atolopko-czi ... I found [this](https://stackoverflow.com/questions/70767090/github-actions-on-push-and-on-pull-request-difference#:~:text=In%20general%2C%20push%20will%20trigger,from%20a%20fork%20for%20example.) which suggests that `pull_request` and `push` are the same unless we're using a fork-repo model (which we aren't)

I'll add a comment to that effect for the benefit of the next person to read through here.

Reply:
GTK! Thanks for the clarifying comment.